TransferMethod/Sources/SelectTransferMethodTypePresenter.swift

```diff
@@ -302,9 +304,12 @@ final class SelectTransferMethodTypePresenter {
         view.transferMethodTypeTableViewReloadData()
     }
 
-    private func trackUILoadImpression() {
+    func trackUILoadImpression() {
+        guard loadedSuccessfully else {
```

Comment:
when `trackUILoadImpression()` is called `loadedSuccessfully` will always be true as per the code, do we need this variable?